homeassistant/components/lacrosse_view/config_flow.py

```diff
@@ -47,8 +45,14 @@ class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
     """Handle a config flow for LaCrosse View."""
 
     VERSION = 1
-    data: dict[str, str] = {}
-    locations: list[Location] = []
+
+    def __init__(self) -> None:
+        """Initialize the config flow."""
+        self.data: dict[str, str] = {}
+        self.locations: list[Location] = []
+        self.reauth = False
+        self.location_id = ""
+        self.location_name = ""
```

Comment:
Consider setting _reauth_data to the input data then have the same state in 1 variable instead of three.